components/canvas/webgl_thread.rs

```diff
@@ -1660,6 +1667,17 @@ impl WebGLImpl {
                     .send(gl.get_tex_parameter_iv(target, param as u32))
                     .unwrap();
             },
+            WebGLCommand::GetInternalFormativ(
+                target,
+                internal_format,
+                param,
+                max_count,
```

Comment:
Rather than having the caller pass the size of the parameter array, we should match on the param and calculate how large the array should be.

Additionally, rather than having the script thread send two commands to retrieve the sample count followed by the samples, we should transparently get the number of required samples when handling the request for samples.